contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -526,15 +525,24 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         return receiptQueues[originBlockchainID].getReceiptAtIndex(index);
     }
 
+    /**
+     * @dev Calculates the message ID for the given source blockchain ID and message nonce.
```

Comment:
can you update comemnt to include destination blockchainID and addres of this instance